kustomize/overlays/prod/secrets/event-bridge-shard-operator-secrets.yaml

```diff
@@ -11,6 +11,7 @@ spec:
     WEBHOOK_CLIENT_SECRET: AgCEYdpBHl7MXg2FnsFdVvfdChoc8fcrKy5dAjXl+VrBnlkAMk6ryIVubkUOi3TZb6b5uh29ViTfUr08xvMmkQF5jxi7wjyiQoS4B4agn+H8YfuI5rWjXFJM5OX58Oi/uXrcWPmq4AX2oJznPT9gI+taqmvG1QlId9jGb46QuE8d27LqzZ6GGIrQCbOrEpajctJMxuMorT4Cx1LqlJD7rM9B40DVUq6gF2XvnUtARbJMxXEsw+hyf8woJzD9K9v7KjL1024ZYFsOFVYxFDr1I+dFrs70UX302f6XGf0SVkwyyyAlFEXLY/p/zIbPKz/pezNut1y/QYwis9P1BbO6yFJt+IB//47w8tlb3iGKlF928nXgTvVRLoegsh/skmMl0BNQCnYxXJS2695fdx1e/oLk2F1lbCgWt4F74UFdTJF+70tZwBRDHelJ/srAIClrfbBr4Q5+O8AMJaRq3tiPFezSyZIpGdL6xyioZR1eRPzc8/HTZKx1TWldDJtid4YId1kXIy+WLvQExrbEPigpDpsXR5pmDPQ9+LFImSNOtnnIFGOATwHzFOu59jzEW7y9IF6e3uy04dm/2D3no4tiuJd+mKg691C7AV2h9sfRv4SD5/Zy/5AhPrU6/92QuB0gFxUVF1Jah1/rNESK0ZSXvLanat9Toboxvn4QBpRWpy+xJ2BbcU/n7cKtHJ6/rfK9IPHtxxjDFLzyRhAe6cDcUfv6bOymM93TuMmDZ97OBg+EDRNZ8/w=
     WEBHOOK_TECHNICAL_ACCOUNT_ID: AgADq6TGw0QVIR1NwRsINHbDx83vrz2foAKXWzpKV+P9AYySCU2FmCn4we4Lkjpy9y5ppaaitYx5cpMwOT7V5oirvpgAIV5HHUyfkGgRhhEo0xnPqLAX91juqpRmz3LSFoGgWU0l9o04lEPynIxoO/I2swg2g2iTSFylSpF4izIgzCr5Qb0So+L3ITqbBsu4DnRK4Dfy0362yvuN63PFqb8hdQAcxU2tAgs2ZDYceWeGZKzVtp5btxQyYtvLzqfwBj+FUUkC5JX/CZzLlrw1kO+/sWm9r/caCfWGhcbAru9GUNmmNYrUukc/b+ACij+AN2c4d5zI09t8ICK2X7pb4p1NpCVSI8wJu4qzVRLY+ugvkOP2KUmO8JEUv7dnIRvcGNpNiWaVlsTui8OzG3OrGogUyKUi9EEIg7T6/KrhOTnTftbyB2s6C6xtD5vwaAb5CqXfJMhxgX7hD1WzZfJRyQfwGNcQCPgIvEW2FeMCOfRu94871c8ni9oG9u5wIRrX+O02Jq2n4HXDZl/B5uS/QuljDcOpZN/TDAK5RnAyiQCDRbtLeIro69h/pPrti+UyH/baVhe8SJwN6UHVw/z6t0K7bRsPTW5/lAoG+lO90uylR3yAJTGDeMWMVixGEzsb/tEUHTX9A14mvSz6iERD/m0xF/WX6i7tvIzpsG1b53j98MyYEwvoM9koOn/bgh4cJh5HLj0dQ/X70Q==
+    EVENT_BRIDGE_SHARD_ID: AgBqr9S/CBggW4OIXgo8wRuF9CpuOsIxPNqaure6XztYmGqNlTTQKrSw0gZQgjy7rDWkffeC8q/nwXsPhaQLU9qaEGzu9mBnQUi7N7aq06QpNx4HLXmDYfhwB97ULqVViRy9tmStVknlL3CAaT/K8oXVoqAcOSB5sSeLDH5eCIyf4iW7PfpOSPK5zZPXoRahd15r/YqGOzLjSsC8KRise6o7oDYdIcBL0GdvrslXuCEF5s9lsMVRe2ZTmKQg9OD2WG+jdgRlS4ryUnhWDgWSpHTo9iCL6vY+kUZ/lTC7gBSxoplxaYWeOGgwfeooFI2Fc2euqi3m8JmKlAQ2FOkC8ij84gHlv7sCoTTA/+HVQwd8yiIDmqFyjneswbWSGTzcjy4motoSnrFxvKYVo3BfO6ypYxMt3sh7T1TtQGgyqOLIyNJlNF5lcDDAIalOwFLure0nDy9yF4SQ/q2BXAGZM/3HxVjdBb/SgJcStJ3/a+3fh0aeN9X3qlhyJLtTqc616cgQEz2BuV9bYfW3Eq5hNPjcteVaXbA6lzZZgFGof6hpW/NK14n3BThMJfvRHc1hwrxvhzRK9kirzra5X3Wuw5TZHNo3pNtPEm+ybqDesjtqEHLl8k2dR4S1pndGTY6Uq5zETOZZjNYyGoEVdqMfkBKQUXJ6SiwOm7pZEv2dDXaaQEPDUCaH3gui1JSCrDedojP/xZ+mU/eJ0L/uXO4ZrIFqxXYleMdvOaE/vYEMRHjkTccjHq0=
```

Comment:
This is outdated. `dev` is now using sso.redhat so you have to encrypt the new shard-id. Ping me on PM to get the new value to encrypt

Reply:
done